plugins/cloud-costs/Dockerfile

```diff
@@ -1,17 +1,24 @@
-FROM gcr.io/google.com/cloudsdktool/google-cloud-cli:473.0.0
-
-RUN apt-get update && \
-    apt-get install -y \
-        python3 \
-        python3-pip \
-        python3-setuptools \
-        groff \
-        less \
-        jq \
-    && pip3 install --upgrade pip \
-    && apt-get clean
-
-RUN pip3 --no-cache-dir install --upgrade awscli
+FROM alpine:3.19
+
+ARG TARGETARCH
+ARG TARGETOS
+
+RUN apk -U upgrade
+RUN apk update && apk upgrade
+RUN apk add jq bash curl moreutils coreutils
+RUN apk add unzip
+
+RUN apk add python3
+
+RUN apk add --no-cache aws-cli
+
+RUN if [ "${TARGETARCH}" = "arm64" ] ; then googleArch="arm"; else googleArch="x86_64"; fi && \
+    curl https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-400.0.0-${TARGETOS}-${googleArch}.tar.gz -o "google-cloud-sdk-400.0.0.tar.gz" && \
```

Comment:
[latest gcloud sdk version is 473](https://cloud.google.com/sdk/docs/install)

Reply:
Yes but that one has more high vulns